cloudinstall/utils.py

```diff
@@ -145,7 +145,37 @@ def sanitize_config_items(_cfg):
         return sanitize_config_items(cfg_cli_opts)
 
 
-def load_charms():
+def load_ext_charms(plug_path, charm_modules):
+    """ Load external charms from plugin directory
+
+    :param plug_path: Top level dir housing 'charms/'
+    :param charm_modules: Existing charms usually import by load_charms()
+    """
+    if os.path.exists(plug_path):
+        try:
+            sys.path.insert(0, plug_path)
+            import charms
+        except ImportError:
+            log.error("No external charms found, not importing.")
+            return charm_modules
```

Comment:
Another point where maybe we should just raise? If someone specifies a plugin path but there's nothing importable there, it seems likely that they intended for something to be imported and should be notified.
What do you think?